dlc-manager/tests/channel_execution_tests.rs

```diff
@@ -101,6 +101,7 @@ enum TestPath {
     RenewReject,
     RenewRace,
     RenewEstablishedClose,
+    CancelOffer,
```

Comment:
If both offer and accept can cancel it might be good to test both cases.

Reply:
see comment above, that was not intentional.